signaturit.js

```diff
@@ -77,7 +77,7 @@ function extractPostParameters (form, files, recipients, parameters)
         );
     });
 
-    recipients = [].concat(recipients);
+    recipients = recipients ? [].concat(recipients) : [];
```

Comment:
Recipients is always required. Can you remove that check, please?

Thanks!

Reply:
Alright, fixed!